gkr/src/main.rs

```diff
@@ -58,106 +51,56 @@ fn main() {
-                "poseidon" => run_benchmark::<M31ExtConfigSha2Orion>(
-                    &args,
-                    Config::new(GKRScheme::GkrSquare, mpi_config.clone()),
-                ),
-                _ => unreachable!("Unsupported PCS type for M31"),
+                "keccak" => {
+                    run_benchmark::<M31ExtConfigSha2OrionVanilla>(&args, mpi_config.clone())
+                }
+                "poseidon" => {
+                    run_benchmark::<M31ExtConfigSha2OrionSquare>(&args, mpi_config.clone())
+                }
+                _ => unreachable!(""),
             },
             _ => unreachable!("Unsupported PCS type for M31"),
         },
         "fr" => match pcs_type {
             PolynomialCommitmentType::Raw => match args.circuit.as_str() {
-                "keccak" => run_benchmark::<BN254ConfigSha2Raw>(
-                    &args,
-                    Config::new(GKRScheme::Vanilla, mpi_config.clone()),
-                ),
-                "poseidon" => run_benchmark::<BN254ConfigSha2Raw>(
-                    &args,
-                    Config::new(GKRScheme::GkrSquare, mpi_config.clone()),
-                ),
+                "keccak" => run_benchmark::<BN254ConfigSha2Raw>(&args, mpi_config.clone()),
                 _ => unreachable!(),
             },
             PolynomialCommitmentType::Hyrax => match args.circuit.as_str() {
-                "keccak" => run_benchmark::<BN254ConfigSha2Hyrax>(
-                    &args,
-                    Config::new(GKRScheme::Vanilla, mpi_config.clone()),
-                ),
-                "poseidon" => run_benchmark::<BN254ConfigSha2Hyrax>(
-                    &args,
-                    Config::new(GKRScheme::GkrSquare, mpi_config.clone()),
-                ),
+                "keccak" => run_benchmark::<BN254ConfigSha2Hyrax>(&args, mpi_config.clone()),
                 _ => unreachable!(),
             },
             PolynomialCommitmentType::KZG => match args.circuit.as_str() {
-                "keccak" => run_benchmark::<BN254ConfigMIMC5KZG>(
-                    &args,
-                    Config::new(GKRScheme::Vanilla, mpi_config.clone()),
-                ),
-                "poseidon" => run_benchmark::<BN254ConfigMIMC5KZG>(
-                    &args,
-                    Config::new(GKRScheme::GkrSquare, mpi_config.clone()),
-                ),
+                "keccak" => run_benchmark::<BN254ConfigMIMC5KZG>(&args, mpi_config.clone()),
                 _ => unreachable!(),
             },
             _ => unreachable!("Unsupported PCS type for BN254"),
         },
         "gf2ext128" => match pcs_type {
             PolynomialCommitmentType::Raw => match args.circuit.as_str() {
-                "keccak" => run_benchmark::<GF2ExtConfigSha2Raw>(
-                    &args,
-                    Config::new(GKRScheme::Vanilla, mpi_config.clone()),
-                ),
-                "poseidon" => run_benchmark::<GF2ExtConfigSha2Raw>(
```

Comment:
I suppose a bunch of GKR square configs are no longer used in benchmarking, namely a bunch of poseidon hashes are no longer supported?